network-area-diagram/src/main/java/com/powsybl/nad/svg/metadata/DiagramMetadata.java

```diff
@@ -106,7 +106,7 @@ public void addNode(String svgId, String equipmentId, String positionX, String p
         nodesMetadata.add(new NodeMetadata(svgId, equipmentId, positionX, positionY));
     }
 
-    public void addEdge(String svgId, String equipmentId) {
-        edgesMetadata.add(new EdgeMetadata(svgId, equipmentId));
+    public void addEdge(String svgId, String equipmentId, String node1DiagramId, String node2DiagramId) {
```

Comment:
You could now rename the `diagramId`s with `svgId`

Reply:
renamed